src/common/workspace-store.ts

```diff
@@ -243,6 +243,7 @@ export class WorkspaceStore extends BaseStore<WorkspaceStoreModel> {
       throw new Error(`workspace ${id} doesn't exist`);
     }
     this.currentWorkspaceId = id;
+    clusterStore.setActive(null);
```

Comment:
We have to reset active cluster if workspace changes. Otherwise UX is really confusing if active cluster is from non-active workspace.

Reply:
I thought we had a PR that saved the previous active workspace's active cluster